### PR TITLE
refactor: remove flow

### PR DESCRIPTION
**Pull request by colincclark**
This PR does https://github.com/newsuk/times-components/issues/1421

- remove flow from the codebase so new starters are not confused (our new direction will be with TypeScript in the future
- amended draft regex which the linter is complaining about in `jest-serializer`

### CODE REVIEW

File: packages/jest-configurator/src/source-loader.js [hunk 1]
```diff
@@ -8,45 +8,30 @@ import { readFileSync } from "fs";
 import { createHash } from "crypto";
 import path from "path";
 
-const readSource = (filename: string): string =>
-  readFileSync(filename).toString();
+const readSource = filename => readFileSync(filename).toString();
```
Comment (nathggns) [inline packages/jest-configurator/src/source-loader.js]: The `// @flow` comments are still here. Might wanna remove them.

**Comment by Belema** [comment]
Running `git grep flow` on this branch returns,
```
.github/CONTRIBUTING.md:* We use [flow](https://flow.org/) and
.github/CONTRIBUTING.md:Avoid this pattern and use control flow to choose a specific `styled-component`
android/app/src/main/java/com/storybooknative/RNTextSizeModule.java:    // https://stackoverflow.com/questions/3654321/measuring-text-height-to-be-drawn-on-canvas-android
android/app/src/main/java/com/storybooknative/RNTextSizeModule.java:     * https://stackoverflow.com/questions/27631736
bitrise.yml:workflows:
packages/ad/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/ad/__tests__/android/__snapshots__/ad-placeholder-with-style.android.test.js.snap:        "overflow": "hidden",
packages/ad/__tests__/ios/__snapshots__/ad-placeholder-with-style.ios.test.js.snap:        "overflow": "hidden",
packages/ad/__tests__/web/__snapshots__/ad-placeholder-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/__snapshots__/ad-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/__snapshots__/ad-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/__snapshots__/ad-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/__snapshots__/ad-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/__snapshots__/ad-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/__snapshots__/ad-with-style.web.test.js.snap:  overflow: hidden;
packages/ad/__tests__/web/ad-placeholder-with-style.web.test.js:  "overflow",
packages/ad/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/ad/src/dom-context.js:            overflow: hidden;
packages/ad/src/dom-context.web.js:        style={{ height: "auto", overflow: "hidden", width }}
packages/ad/src/styles/index.js:    overflow: "hidden"
packages/article-byline/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-byline/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/article-flag/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-flag/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/article-image/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-image/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/article-label/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-label/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/article-list/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-list/__tests__/web/__snapshots__/article-list-error-with-styles.web.test.js.snap:  overflow: hidden;
packages/article-summary/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-summary/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/article-topics/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article-topics/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/article/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/article/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/author-profile/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/author-profile/__tests__/android/__snapshots__/author-profile-with-style.android.test.js.snap:            "overflow": "hidden",
packages/author-profile/__tests__/android/__snapshots__/author-profile-with-style.android.test.js.snap:          "overflow": "hidden",
packages/author-profile/__tests__/ios/__snapshots__/author-profile-with-style.ios.test.js.snap:            "overflow": "hidden",
packages/author-profile/__tests__/ios/__snapshots__/author-profile-with-style.ios.test.js.snap:          "overflow": "hidden",
packages/author-profile/__tests__/shared-with-style.web.js:  "overflow",
packages/author-profile/__tests__/web/__snapshots__/author-profile-with-style.web.test.js.snap:  overflow: hidden;
packages/author-profile/src/styles/shared.js:    overflow: "hidden",
packages/author-profile/src/styles/shared.js:    overflow: "hidden",
packages/brightcove-video/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/button/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/button/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/caption/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/caption/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/card/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/card/__tests__/web/__snapshots__/card-loading-style.test.js.snap:  overflow: hidden;
packages/card/__tests__/web/__snapshots__/card-loading-style.test.js.snap:  overflow: hidden;
packages/card/__tests__/web/__snapshots__/card-render-style.test.js.snap:  overflow: hidden;
packages/card/__tests__/web/__snapshots__/card-reverse-style.test.js.snap:  overflow: hidden;
packages/card/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/date-publication/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/date-publication/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/depend/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/error-view/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/error-view/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/eslint-config-thetimes/index.js:    "plugin:flowtype/recommended",
packages/eslint-config-thetimes/index.js:  plugins: ["flowtype"],
packages/eslint-config-thetimes/package.json:    "eslint-plugin-flowtype": "2.41.0",
packages/gestures/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/gestures/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/gradient/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/gradient/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/icons/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/icons/package.json:    "flow": "node_modules/flow-bin/cli.js",
packages/icons/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/image/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/image/__tests__/shared-with-style.web.js:  "overflow",
packages/image/__tests__/web/__snapshots__/image-with-style.web.test.js.snap:  overflow: hidden;
packages/image/__tests__/web/__snapshots__/image-with-style.web.test.js.snap:  overflow: hidden;
packages/image/image.showcase.js:              overflow: "hidden",
packages/image/image.showcase.js:              overflow: "hidden",
packages/image/image.showcase.js:              overflow: "hidden",
packages/image/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/image/src/image.web.js:        overflow: "hidden",
packages/jest-configurator/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/jest-configurator/src/source-loader.js:// @flow
packages/jest-serializer/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/link/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/link/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/markup/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/markup/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/pagination/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/pagination/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/provider-test-tools/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/provider-test-tools/fixtures/article.json:            "value": "The “elite” have been chosen from those who, through his Heaven SX website and social media, apply for invitations to his parties. Most applicants, he claims, are discarded because of paunch and age, and only the best-looking get to pay around £100 for a ticket. They drink, chat and then, at midnight, the women get changed into lingerie. “It’s loads of girls, all giggling, putting on their stockings, hair flowing,” he explains. “Then they walk in, the guys go, ‘Wow,’ and the mood changes. They all start disappearing and, before you know it, there are 30 girls on a bed and designer gear everywhere. When I pull it off I can’t believe I created it.”"
packages/provider-test-tools/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/provider/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/pull-quote/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/pull-quote/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/related-articles/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/related-articles/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/slice/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/slice/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/storybook/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/storybook/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/styleguide/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/styleguide/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/svgs/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/svgs/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/tealium-utils/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/tealium-utils/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/tealium/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/tealium/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/topic/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/utils/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/utils/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/video-label/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/video-label/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/video/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
packages/video/video.showcase.js:            overflow: "hidden",
packages/watermark/.babelrc:    ["flow-react-proptypes", { "ignoreNodeModules": true }],
packages/watermark/package.json:    "babel-plugin-flow-react-proptypes": "22.0.0",
yarn.lock:"@babel/plugin-syntax-flow@^7.0.0":
yarn.lock:  resolved "https://registry.yarnpkg.com/@babel/plugin-syntax-flow/-/plugin-syntax-flow-7.0.0.tgz#70638aeaad9ee426bc532e51523cff8ff02f6f17"
yarn.lock:"@babel/plugin-transform-flow-strip-types@^7.0.0-beta":
yarn.lock:  resolved "https://registry.yarnpkg.com/@babel/plugin-transform-flow-strip-types/-/plugin-transform-flow-strip-types-7.0.0.tgz#c40ced34c2783985d90d9f9ac77a13e6fb396a01"
yarn.lock:    "@babel/plugin-syntax-flow" "^7.0.0"
yarn.lock:    ast-types-flow "0.0.7"
yarn.lock:ast-types-flow@0.0.7:
yarn.lock:  resolved "https://registry.yarnpkg.com/ast-types-flow/-/ast-types-flow-0.0.7.tgz#f70b735c6bca1a5c9c22d982c3e39e7feba3bdad"
yarn.lock:    ast-types-flow "0.0.7"
yarn.lock:babel-plugin-flow-react-proptypes@22.0.0:
yarn.lock:  resolved "https://registry.yarnpkg.com/babel-plugin-flow-react-proptypes/-/babel-plugin-flow-react-proptypes-22.0.0.tgz#d93ad6007c0ea6e4e90b6437dd446e5d9c2d8cdf"
yarn.lock:babel-plugin-syntax-flow@^6.18.0, babel-plugin-syntax-flow@^6.5.0, babel-plugin-syntax-flow@^6.8.0:
yarn.lock:  resolved "https://registry.yarnpkg.com/babel-plugin-syntax-flow/-/babel-plugin-syntax-flow-6.18.0.tgz#4c3ab20a2af26aa20cd25995c398c4eb70310c8d"
yarn.lock:babel-plugin-transform-flow-strip-types@^6.21.0, babel-plugin-transform-flow-strip-types@^6.22.0, babel-plugin-transform-flow-strip-types@^6.5.0, babel-plugin-transform-flow-strip-types@^6.8.0:
yarn.lock:  resolved "https://registry.yarnpkg.com/babel-plugin-transform-flow-strip-types/-/babel-plugin-transform-flow-strip-types-6.22.0.tgz#84cb672935d43714fdc32bce84568d87441cf7cf"
yarn.lock:    babel-plugin-syntax-flow "^6.18.0"
yarn.lock:    babel-plugin-syntax-flow "^6.8.0"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.8.0"
yarn.lock:babel-preset-flow@^6.23.0:
yarn.lock:  resolved "https://registry.yarnpkg.com/babel-preset-flow/-/babel-preset-flow-6.23.0.tgz#e71218887085ae9a24b5be4169affb599816c49d"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.22.0"
yarn.lock:    babel-plugin-syntax-flow "^6.5.0"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.5.0"
yarn.lock:    babel-plugin-syntax-flow "^6.5.0"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.5.0"
yarn.lock:    babel-plugin-syntax-flow "^6.5.0"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.5.0"
yarn.lock:    babel-preset-flow "^6.23.0"
yarn.lock:eslint-plugin-flowtype@2.41.0:
yarn.lock:  resolved "https://registry.yarnpkg.com/eslint-plugin-flowtype/-/eslint-plugin-flowtype-2.41.0.tgz#fd5221c60ba917c059d7ef69686a99cca09fd871"
yarn.lock:    ast-types-flow "0.0.7"
yarn.lock:flow-bin@0.63.1:
yarn.lock:  resolved "https://registry.yarnpkg.com/flow-bin/-/flow-bin-0.63.1.tgz#ab00067c197169a5fb5b4996c8f6927b06694828"
yarn.lock:flow-parser@^0.*:
yarn.lock:  resolved "https://registry.yarnpkg.com/flow-parser/-/flow-parser-0.81.0.tgz#cda0ce856b8b87bc32790c02abd039d5048a4711"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.22.0"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.8.0"
yarn.lock:    flow-parser "^0.*"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.8.0"
yarn.lock:    flow-parser "^0.*"
yarn.lock:    "@babel/plugin-transform-flow-strip-types" "^7.0.0-beta"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.21.0"
yarn.lock:    babel-plugin-transform-flow-strip-types "^6.21.0"
```

File: packages/jest-configurator/src/source-loader.js (hunk 1 above)
Comment (colincclark) [reply packages/jest-configurator/src/source-loader.js]: Done

**Comment by Belema** [comment]
@colincclark, lint failing on bitrise...

**Comment by colincclark** [comment]
This requires more work than I have time for now

**Pull request closed without merging**